enterprise/api/v1/serializers.py

```diff
@@ -647,9 +647,10 @@ def validate_tpa_user_id(self, value):
         It first uses the third party auth api to find the associated username to do the lookup.
         """
         enterprise_customer = self.context.get('enterprise_customer')
+        request_user = self.context.get('request_user')
 
         try:
-            tpa_client = ThirdPartyAuthApiClient()
+            tpa_client = ThirdPartyAuthApiClientJwt(request_user)
```

Comment:
what will happen if we do not have `request_user` in context for EnterpriseCustomerCourseEnrollmentsSerializer?

Reply:
I cross-checked it. The view that is using `EnterpriseCustomerCourseEnrollmentsSerializer` already passes `request_user` in context. See this for reference: https://github.com/edx/edx-enterprise/blob/master/enterprise/api/v1/views.py#L178

Reply:
yes, We are passing for this instance, but what if in future someone instantiates `EnterpriseCustomerCourseEnrollmentsSerializer` object and do not pass `request_user`

Reply:
I can pass `None` as a default value and then make the changes accordingly. Would that solve the problem? If you have any other solutions that you can propose, please do.

Reply:
yes,
we can put these lines in try-catch.
```
enterprise_customer = self.context['enterprise_customer']
request_user = self.context['request_user']
```
and catch a KeyError expection then after logging error we can return None.